torchbenchmark/__init__.py

```diff
@@ -14,6 +15,29 @@
 model_dir = 'models'
 install_file = 'install.py'
 
+# Enum class to hold all the available domain names
+class Domain(Enum):
+    COMPUTER_VISION = "computer vision"
+    OTHER_COMPUTER_VISION = "other computer vision"
+    NLP = "natural language processing"
+    SPEECH = "speech"
+    RECOMMENDATION = "recommendation"
+    REINFORCEMENT_LEARNING = "reinforcement learning"
+    OTHER = "other"
+
+# Enum class to hold all the available task names
+class Task(Enum):
+    SEGMENTATION = "segmentation"
+    CLASSIFICATION = "classification"
+    DETECTION = "detection"
+    BACKGROUND_MATTING = "Background_Matting"
```

Comment:
better to be consistent (in domain you used spaces and lowercase) - i think spaces and lowercase is better, it's also easy for someone to add underscores later programmatically if they want them)